roles/zookeeper/defaults/main.yml

```diff
@@ -1,69 +1,19 @@
 ---
-zookeeper_version: 3.4.11
-## Service options
-
-# Owner
-zookeeper_user: zookeeper
-zookeeper_group: zookeeper
-
-# start on boot
-zookeeper_service_enabled: true
-zookeeper_service_state: started
-
-# Files & Paths
-zookeeper_data_dir: /var/lib/zookeeper
-zookeeper_log_dir: /var/log/zookeeper
-zookeeper_dir: /usr/share/zookeeper
-zookeeper_conf_dir: /etc/zookeeper
-
-# Connection properties
-client_port: 2181
-init_limit: 5
-sync_limit: 2
-tick_time: 2000
-zookeeper_client_port: '{{ client_port }}'
-zookeeper_init_limit: '{{ init_limit }}'
-zookeeper_sync_limit: '{{ sync_limit }}'
-zookeeper_tick_time: '{{ tick_time }}'
-
-zookeeper_autopurge_purgeInterval: 0
-zookeeper_autopurge_snapRetainCount: 10
-
-# Java options
-zookeeper_jmx_enabled: true
-zookeeper_jmx_port: 1099
-zookeeper_java_opts: '-Dcom.sun.management.jmxremote.authenticate=false \
-                      -Dcom.sun.management.jmxremote.ssl=false \
-                      -Dcom.sun.management.jmxremote.local.only=$JMXLOCALONLY \
-                      -Dcom.sun.management.jmxremote.port={{ zookeeper_jmx_port }} \
-                      -Djava.rmi.server.hostname={{ ansible_nodename }}'
-
-# Rolling file appender setttings
-zookeeper_rolling_log_file_max_size: 10MB
-zookeeper_max_rolling_log_file_count: 10
-
-# List of dicts (i.e. {zookeeper_hosts:[{host:,id:},{host:,id:},...]})
-zookeeper_hosts_dict:
-  - host: '{{ lib_zk1_host_name }}'  # The leader in the Zookeeper ensemble
+# defaults file for roles/zookeeper
+zk_version: "3.4.13"
```

Comment:
Are we controlling the zookeeper version? or just installing whatever the latest package version is? We probably need to think about this for the weekly package updates - could an upgrade break something?

Reply:
I also think this can go. We were considering the need of building zookeeper from source.